main.py

```diff
@@ -21,7 +24,23 @@
 
 PRODUCTION = False # added for convenience, all factories eventually created in main and passed to trade_bot
 
+# SERVER SETUP
+def flask_init():
+    endpoints = []
```

Comment:
Can we define endpoint inside of the WebService class? Its hard to track what they do by finding their labels here and then having to go WebService to discover what they are calling.

Reply:
See example here, I used your code: 

https://github.com/BensGitHub2022/MT5-Expert-Advisor/blob/ben/webservice_example/src/web_service/web_service.py